Expand the reference for the TypeScript library (#15361)
digital-asset/daml: docs/source/app-dev/bindings-ts/index.rst

.. _typescript:

> The JavaScript client libraries allow to easily build frontend applications that interact with the :ref:`HTTP JSON API service <json-api>`.

Reviewer: ```suggestion
The JavaScript client libraries allow you to easily build frontend applications that interact with the :ref:`HTTP JSON API service <json-api>`.
```

> To quickly build a prototype or an application with relatively simple requirements, these libraries can dramatically reduce the time
> necessary to develop a full-stack application by abstracting away implementation details.

Reviewer: ```suggestion
These libraries can dramatically reduce the time
necessary to develop a full-stack application by abstracting away implementation details, particularly when building a prototype or an application with relatively simple requirements.
```

> The `@daml/types <ts-daml-types_>`_ library contains the TypeScript data types corresponding to primitive Daml data type, such as ``Party`` or ``Text``.

Reviewer: ```suggestion
The `@daml/types <ts-daml-types_>`_ library contains the TypeScript data types corresponding to primitive Daml data types, such as ``Party`` or ``Text``.
```
